src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -11,15 +12,35 @@
 public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
+    public static final int PAGE_CONTAIN = 3;
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Displays all persons in the address book as a list with index numbers.\n"
-            + "Example: " + COMMAND_WORD;
+            + ": Displays all persons or persons in one page in the address book as a list with index numbers.\n"
+            + "Example: " + COMMAND_WORD + "\n"
+            + "Example: " + COMMAND_WORD + " 1";
 
+    public ListCommand(int targetPage) { super(targetPage); }
+    public ListCommand() {}
 
     @Override
     public CommandResult execute() {
         List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
-        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
+        final int targetPage = getTargetIndex();
+        if(targetPage > 0) {
```

Comment:
The placing of space seems to be a bit inconsistent; you've missed out a space after the `if`.